.github/workflows/deploy-staging-and-prepare-release.yaml

```diff
@@ -61,6 +61,7 @@ jobs:
       IS_CLOUD_INSTANCE: true
       POSTHOG_KEY: ${{ secrets.POSTHOG_KEY }}
       POSTHOG_HOST: ${{ secrets.POSTHOG_HOST }}
```

Comment:
I'm not sure why we removed the secret from other places but not this one? Is it only needed for this one or it's a leftover?

(I'm assuming it's a frontend/client-side secret and not backend secret as well?)